python-client/giskard/scanner/templates/_issues_table.html

```diff
@@ -12,19 +12,20 @@
                 </td>
                 <td class="p-3">
                     <code class="mono text-blue-300">
-                    {{issue.slice_fn|replace("&", "<br>")}}
+                    {{issue.domain|replace("&", "<br>")}}
                 </code>
                 </td>
                 <td class="p-3">
-                    {{issue.test_name}}
+                    {{ issue.metric }}
                 </td>
                 <td class="p-3">
-                    <span class="{{'text-red-400' if issue.is_major else 'text-amber-200'}}">{{"{:.2f}".format(issue.test_results.metric * 100) }}% than
-                        global</span>
+                    <span class="{{'text-red-400' if issue.is_major else 'text-amber-200'}}">
+                        {{ issue.deviation }}
+                    </span>
                 </td>
                 <td class="p-3">
                     <span class="text-gray-400">
-                        {{issue.test_results.actual_slices_size[0]}} samples ({{ '{:.2f}'.format(100 * issue.test_results.actual_slices_size[0] / issue.test_results.reference_slices_size[0])}}%)
+                        {{ issue.description }}
```

Comment:
we should be careful displaying user-provided content, for the XSS reasons. There's an escaping mechanism (`|e`) like : `{{ dangerous|e }}`

Reply:
Autoescaping is enabled for HTML but a bug in the iframe generation prevented this from being effective. It is now fixed.